paths_cli/tests/compiling/test_networks.py

```diff
@@ -86,17 +34,50 @@ def test_build_tps_network(cv_and_states):
     dct = yaml.load(yml, yaml.FullLoader)
     compiler = {
         'volume': mock_compiler('volume', named_objs={"A": state_A,
-                                                    "B": state_B}),
+                                                      "B": state_B}),
     }
     with mock.patch.dict(_COMPILERS_LOC, compiler):
-        network = build_tps_network(dct)
+        network = TPS_NETWORK_PLUGIN(dct)
     assert isinstance(network, paths.TPSNetwork)
     assert len(network.initial_states) == len(network.final_states) == 1
     assert network.initial_states[0] == state_A
     assert network.final_states[0] == state_B
 
-def test_build_mistis_network():
-    pytest.skip()
 
-def test_build_tis_network():
-    pytest.skip()
+def test_build_mistis_network(cv_and_states, unidirectional_tis_compiler):
+    cv, state_A, state_B = cv_and_states
+    mistis_dict = {
+        'interface_sets': [{
+            'initial_state': "A",
+            'final_state': "B",
+            'cv': 'cv',
+            'minvals': 'float("-inf")',
+            'maxvals': "np.array([0, 0.1, 0.2]) * np.pi"
+        }]
+    }
```

Comment:
I would feel more comfortable if this tested an actual MISTIS setup instead of mimicing the TIS setup, maybe
```suggestion
    mistis_dict = {
        'interface_sets': [{
            'initial_state': "A",
            'final_state': "B",
            'cv': 'cv',
            'minvals': 'float("-inf")',
            'maxvals': "np.array([0, 0.1, 0.2]) * np.pi"
        }]*2
    }
```
Feel free to ignore

Reply:
Went with a more complete treatment in [a3e95c3](https://github.com/openpathsampling/openpathsampling-cli/pull/86/commits/a3e95c3386f316fd1088b528e5adf7c287a729c9)